tasks/sep/006-1-sc-changes/SignFromJson.s.sol

```diff
@@ -142,14 +151,14 @@ contract SignFromJson is OriginalSignFromJson {
     }
 
     function checkSemvers() internal view {
-        // These are the expected semvers based on the `op-contracts/v1.3.0` release.
-        // https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2fv1.3.0
+        // These are the expected semvers based on the `op-contracts/v1.4.0-rc.4` release.
+        // https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.4.0-rc.4
```

Comment:
Should we be using [v1.5.0](https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.5.0-rc.1) here?

Reply:
oh, I see, these are contracts released in v1.4.0